wagtail/admin/views/generic/mixins.py

```diff
@@ -107,8 +108,10 @@ def form_valid(self, form):
 class LocaleMixin:
     @cached_property
     def i18n_enabled(self) -> bool:
-        return getattr(settings, "WAGTAIL_I18N_ENABLED", False) and issubclass(
-            self.model, TranslatableMixin
+        return (
+            (model := getattr(self, "model", None))
```

Comment:
This is probably too much, so feel free to remove

Reply:
Hmm, the original logic before f6fef0cf69e099aeb1c5d747608e28a208a167aa would probably allow `model` to be `None`, whereas now it would fail when accessing `self.i18n_enabled` due to `issubclass(None, TranslatableMixin)`. I'd say we can keep this, but maybe do the settings check first and then explicitly checking that the model is not None, e.g.

```py
    @cached_property
    def i18n_enabled(self) -> bool:
        return (
            getattr(settings, "WAGTAIL_I18N_ENABLED", False)
            and (model := getattr(self, "model", None)) is not None
            and issubclass(model, TranslatableMixin)
        )
```

so that it always returns boolean (otherwise it might return a model class due to short-circuiting).